package.json

```diff
@@ -115,41 +115,42 @@
     "react-native-android-location-services-dialog-box": "^2.8.2",
     "react-native-animatable": "^1.3.3",
     "react-native-ble-plx": "2.0.1",
-    "react-native-camera": "3.19.0",
-    "react-native-config": "0.11.7",
+    "react-native-camera": "3.43.6",
+    "react-native-config": "1.4.2",
     "react-native-crypto": "^2.2.0",
     "react-native-easy-markdown": "^2.0.0",
     "react-native-extra-dimensions-android": "^1.2.5",
-    "react-native-fingerprint-scanner": "^4.0.0",
-    "react-native-gesture-handler": "^1.6.0",
-    "react-native-keychain": "^5.0.0",
+    "react-native-fingerprint-scanner": "^6.0.0",
+    "react-native-gesture-handler": "^1.10.3",
+    "react-native-keychain": "^7.0.0",
     "react-native-level-fs": "^3.0.0",
     "react-native-locale": "0.0.19",
-    "react-native-modal": "^11.5.4",
+    "react-native-modal": "^11.10.0",
     "react-native-navigation-bar-color": "^2.0.1",
     "react-native-os": "^1.2.6",
-    "react-native-progress": "^4.0.3",
-    "react-native-qrcode-svg": "6.0.3",
-    "react-native-randombytes": "^3.6.0",
-    "react-native-reanimated": "^1.10.1",
-    "react-native-redash": "^15.11.1",
-    "react-native-safe-area-context": "^0.7.3",
+    "react-native-pager-view": "^5.1.5",
```

Comment:
Newly added due to the latest react-native-tab-view](https://github.com/satya164/react-native-tab-view) requirement.